src/test/it/essentials.yml

```diff
@@ -0,0 +1,77 @@
+---
+packaging:
+  configFile: "packager-config.yml"
+  archiveArtifacts: true
+  config:
+    bundle:
+      groupId: "io.jenkins-ci.main"
+      artifactId: "jenkins-war"
+      vendor: "Jenkins project"
+      title: "Jenkins WAR - Stapler Integration Tests"
+      description: "Bundles stapler "
+    war:
+      groupId: "org.jenkins-ci.main"
+      artifactId: "jenkins-war"
+      source:
+        version: 2.107.3
+    plugins:
+      # Includes stapler-adjunct-jquery, we may want to test compatibility
+      - groupId: "org.jenkins-ci.plugins"
+        artifactId: "jquery"
+        source:
+          version: "1.12.4-0"
+      # Includes stapler-groovy, we may want to test compatibility
+      - groupId: "org.jenkins-ci.plugins.icon-shim"
+        artifactId: "icon-shim"
+        source:
+          version: "2.0.3"
+      # Includes Stapler directly
+      #TODO: Why?
+      - groupId: "org.jenkins-ci.plugins"
+        artifactId: "pipeline-maven"
+        source:
+          version: "3.5.5"
+      # Upper bounds and JENKINS-51224
+      - groupId: "org.jenkins-ci.plugins"
+        artifactId: "junit"
+        source:
+          version: "1.20"
+    libPatches:
+      - groupId: "org.kohsuke.stapler"
```

Comment:
Instead of that, we need to rebuild the core with `stapler.version`. CWP needs to be patched to support that.

```yaml
    war:
      groupId: "org.jenkins-ci.main"
      artifactId: "jenkins-war"
      source:
        version: 2.107.3
      libraries:
      -  property: "stapler.version"
         groupId: "org.kohsuke.stapler"
         artifactId: "stapler"
         source:
           dir: ../../..
```